UnityProject/Assets/Scripts/Learning/ProtipObjectTypes/ProtipObjectOnPickupTrait.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Remoting.Messaging;
+using UnityEngine;
+
+namespace Learning.ProtipObjectTypes
+{
+	public class ProtipObjectOnPickupTrait : ProtipObject
+	{
+		public List<TraitProtips> protipsForTraits;
+
+		/// <summary>
+		/// Dictionaries are not supported in the unity editor for some reason.
```

Comment:
except they are. Use a ``SerializableDictionary<K,V>``

Reply:
noice, adding it soon.

Reply:
done